lib/framework/elastic_connector_actions.rb

```diff
@@ -55,7 +56,9 @@ def self.claim_job(connector_package_id)
         :doc => {
           :sync_now => false,
           :sync_status => Connectors::SyncStatus::IN_PROGRESS,
-          :last_synced => Time.now
+          :scheduling => {
+            :last_synced => Time.now # TODO not sure if this is the right place to set this
+          }
```

Comment:
How does it affect this PR?  Right now it looks out of scope of this PR

Reply:
If it's not updated, the next cycle picks it up to sync again, thinking "oh there's no sync date, so it's a first sync".